cmd/kaspawallet/send.go

```diff
@@ -35,7 +35,13 @@ func send(conf *sendConfig) error {
 
 	var sendAmountSompi uint64
 	if !conf.IsSendAll {
-		sendAmountSompi = uint64(conf.SendAmount * constants.SompiPerKaspa)
+		parsedAmountSompi, err := utils.KasToSompi(conf.SendAmount)
+
+		if err != nil {
+			return err
+		}
+
+		sendAmountSompi = parsedAmountSompi
```

Comment:
Is there a reason it is not directly `sendAmountSompi, err := utils.KasToSompi(conf.SendAmount)`?

Reply:
Yeah, because of shadowing: https://go.dev/play/p/vAX1AHDpmvs

However, we since `err` is already declared above the lines, I can do (and have done):

```
sendAmountSompi, err = utils.KasToSompi(conf.SendAmount)
```

(note that it's `=` and not `:=`)